pkgs/tools/backup/duply/default.nix

```diff
@@ -4,11 +4,11 @@
 
 stdenv.mkDerivation rec {
   pname = "duply";
-  version = "2.3";
+  version = "2.3.1";
 
   src = fetchurl {
     url = "mirror://sourceforge/project/ftplicity/duply%20%28simple%20duplicity%29/2.3.x/duply_${version}.tgz";
-    sha256 = "sha256-I1KkblFnZVOCvcWEarGsSXnzlod2+yZ4okaTpckLFbE=";
+    sha256 = "sha256:149hb9bk7hm5h3aqf19k37d0i2jf0viaqmpq2997i48qp3agji7h";
```

Comment:
I haven't seen this "sha256:" form before. I see a bit of it in nixpkgs, but not as much as "sha256-" form (SRI) or the non-prefixed hash format. Why did you decide to change the form? Where is it documented?

Reply:
```suggestion
    sha256 = "149hb9bk7hm5h3aqf19k37d0i2jf0viaqmpq2997i48qp3agji7h";
```

Reply:
> I haven't seen this "sha256:" form before.

It is what old daemons output when you have a hash mismatch. Normally everyone copies it without the hash prefix.

> Why did you decide to change the form?

I have a wild guess: just copied it. Not a deal breaker and does not matter to much tbh.

> Where is it documented?

I don't know right now and to lazy to search for.